ebean-ddl-generator/src/main/java/io/ebeaninternal/dbmigration/model/MTable.java

```diff
@@ -574,11 +581,11 @@ public MColumn addColumnScalar(String dbColumn, String columnDefn) {
   /**
    * Add a 'new column' to the AddColumn migration object.
    */
-  private void diffNewColumn(MColumn newColumn) {
+  private void diffNewColumn(MColumn newColumn, MTable newTable) {
     if (addColumn == null) {
       addColumn = new AddColumn();
       addColumn.setTableName(name);
-      if (withHistory) {
+      if (newTable.isWithHistory()) {
```

Comment:
We must enable history only, when the new table has history.
if you disable history, then the old table has history, but not the new one